packages/ui/src/components/textarea.tsx

```diff
@@ -0,0 +1,21 @@
+import { forwardRef, TextareaHTMLAttributes } from 'react'
+
+import { cn } from '@utils/cn'
+
+export interface TextareaProps extends TextareaHTMLAttributes<HTMLTextAreaElement> {}
```

Comment:
Not an issue, but if nothing is being added, then `type` is a better choice than `interface`
```suggestion
export type TextareaProps = TextareaHTMLAttributes<HTMLTextAreaElement>
```